### DISCUSSION TITLE
Wrong value

### DISCUSSION THREAD

**Discussion by svenschedlbauer** [jomjol/AI-on-the-edge-device#2459] [Q&A]
Hi @ll, 
I have installed now an ESPcam for our gas, but I get the wrong values and don't know why... :-(
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/110197932/365080c6-dd0d-491d-b074-821a0dff1b78)
I have already tried the 4 different digit models, but no one of the is showing me the correct value. 
Hope anybody can help me.

**Comment by friedpa** [comment]
Have you read the Wiki, how to setup the picture..... first: all numbers on a straight line. Just use the Rotate function and then have a close look, how the frames around the digits must be configured. Tipp: use a diffuser for the flashlight to get less reflections

**Comment by svenschedlbauer** [comment]
Hi friedpa,
Yes, I have done. I already use an AI on the edge for my watermeter, which is running well. The problem here is the digit model in the configuration. I can choose between 4. But none of them show me the correct value. 
Have you an idea for a diffuser for the esp32 cam?

**Comment by svenschedlbauer** [comment]
Is there any possibility to download somewhere an other digit model, which can read the digits better?

**Reply by ghost** [reply]
You can collect your own images ( https://jomjol.github.io/AI-on-the-edge-device-docs/collect-new-images/) and learn your own model (https://jomjol.github.io/AI-on-the-edge-device-docs/Learn-models-with-your-own-images/ ) or share it to us.

We have already a few lcd images. But the lcd has often reflections and less contrast. So it can help to use learning on your own images.
